libs/features/icons/icons.js

```diff
@@ -63,12 +63,12 @@ export default async function loadIcons(icons, config) {
     const parent = icon.parentElement;
     if (parent.childNodes.length > 1) {
       if (parent.lastChild === icon) {
-        icon.classList.add('margin-left');
+        icon.classList.add('margin-inline-start');
       } else if (parent.firstChild === icon) {
-        icon.classList.add('margin-right');
+        icon.classList.add('margin-inline-end');
         if (parent.parentElement.tagName === 'LI') parent.parentElement.classList.add('icon-list-item');
       } else {
-        icon.classList.add('margin-left', 'margin-right');
+        icon.classList.add('margin-inline-start', 'margin-inline-end');
```

Comment:
I think these have already been updated, maybe rebase would eliminate this from the changes https://github.com/adobecom/milo/blob/stage/libs/features/icons/icons.js#L66-L71

Reply:
That did it 👍